frontend/src/locales/de.json

```diff
@@ -115,6 +115,21 @@
         },
         "title": "Einstellungen"
       },
+      "collaboratorListItem": {
+        "cannotAssignAnotherRoleToLastManager": "Du kannst dem letzten Manager keine andere Rolle zuteilen.",
+        "cannotRemoveLastManager": "Du kannst den letzten Manager nicht entfernen",
+        "deactivate": "Deaktivieren",
+        "resendEmail": "E-Mail erneut senden"
+      },
+      "collaboratorListItemDeactivate": {
+        "deactivate": "Deaktivieren",
+        "title": "Mitarbeiter:in deaktivieren",
```

Comment:
Merci fürs gendern. Müssten wir wahrscheinlich mal noch konsequenter umsetzen in den Übersetzungen.

Mitarbeiter:in assoziert für mich irgendwie gleich "Angestellte". Ist das nur meine Assoziation? Oder wollen wir uns auf einen anderen Namen einigen? Mitleitende? Mitglieder?